src/components/wish-results.jsx

```diff
@@ -2,10 +2,24 @@ import React, { Component } from 'react'
 import { Container, Row, Col } from 'reactstrap';
 import WishItem from './wish-item'
 import WishItemSingle from './wish-item-single'
+import characters from '../data/characters.json'
 export default class WishResults extends Component {
   isNewItem(key) {
     return !this.props.inventory[key]
   }
+  getPercentX(item) {
+    if (item.type === 'character') {
+      let matches = characters.filter(c => c.name === item.name);
+      if (matches.length === 1) {
+        return matches[0].percentX || 50;
+      } else {
+        console.log('Cannot find character '+item.name+' in characters.json');
+        return 50;
+      }
+    } else {
+      return 50;
+    }
+  }
```

Comment:
I don't like the idea that every time we get a character in the banner, we have to iterate through a list of a bunch of characters to find the matching one. It would be much more efficient if we just add properties on characters in the banners.
You will have to update the schema validation objects to account for the percentX property